docs/modules/ROOT/examples/example-history-server.yaml

```
@@ -0,0 +1,29 @@
+---
+apiVersion: spark.stackable.tech/v1alpha1
+kind: SparkHistoryServer
+metadata:
+  name: spark-history
+spec:
+  image:
+    productVersion: 3.3.0
+    stackableVersion: 0.3.0
+  logFileDirectory:  # <1>
+    s3:
+      prefix: eventlogs/  # <2>
+      bucket:  # <3>
+        inline:
+          bucketName: spark-logs
+          connection:
+            inline:
+              host: test-minio
+              port: 9000
+              accessStyle: Path
+              credentials:
+                secretClass: s3-credentials-class
```

I would change the credentials class here from `s3-credentials-class` to `history-credentials-class`, like we have in the integration test, so it's more obvious that trying to use _those_ credentials in the SparkApplication app will just be ignored.
```suggestion
                secretClass: history-credentials-class
```